src/main/webapp/matches.html

```diff
@@ -3,11 +3,81 @@
   <head>
     <meta charset="UTF-8">
     <title>Friends of Friends</title>
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>Matches</h1>
+    <div class="container-fluid">
+      <nav class="navbar navbar-expand-sm justify-content-center navbar-background">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <img src="images/logo.jpg" class="rounded-circle" alt="logo" height="80" width="100">
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="profile.html">Profile</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="feed.html">Feed</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="matches.html">Matches</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="content">
+        <h1>Matches</h1>
+        <div class="row row-cols-5">
+          <div class="col card card-container m-5">
```

Comment:
Are these placeholders going to be populated in JS later? Assuming yes, you might as well construct them with JS now and just replace the data source later?

(it's probably better to support a dynamic number rather than assuming there are 5, anyway)

Reply:
yep- the 5 cards right now are just to see and visualize what the page will look like. For now I am focusing on finishing the profile page and slideshow page, and then after the mvp focus on matches page. This is just to kinda visualize what it should look like after the functionality is implemented.